src/backend/joanie/core/serializers.py

```diff
@@ -248,6 +251,7 @@ class OrderLiteSerializer(serializers.ModelSerializer):
         read_only=True,
     )
     enrollments = serializers.SerializerMethodField(read_only=True)
+    organization = serializers.SlugRelatedField(read_only=True, slug_field="id")
```

Comment:
What do you think to use `OrganizationSerializer` here ?

Reply:
I'm sure if it's needed but the present field allowed to fufill all the existing tests :man_shrugging: 
Chaning to a nested field requires a refactoring to allow creating an order via the API. So this should be a separate PR.

Reply:
Ok let's go with that then we will update this aspect when the need will be here.